sdk/output/builtins.go

```diff
@@ -165,6 +186,17 @@ var Nanoseconds = Output{
 	},
 }
 
+// Number is an output type for some number of things. This output has
+// no unit.
+//
+// An example of what a number could be include: an index into something,
+// the number of rows/columns of a table, etc.
+var Number = Output{
+	Name:      "number",
```

Comment:
Is this an integer, unsigned, float, or complex number? We may want something more specific here. I just see confusion coming up later especially if this is used by external organizations.

Reply:
The idea was that it could be any kind of numeric value, without any restriction on the actual typing of it, since reading values are all just `interface{}` types in  the SDK anyway. I think its okay since its just the definition of the output model, and it doesn't place restrictions on the value itself, but I see what you mean that there could be confusion about this later on.

I'll add a note to the comment for it, but we should also probably have a conversation  at some point about outputs and how specific we want to get with them, etc.

Reply:
Makes sense to me. Thanks!